nestkernel/event.h

```diff
@@ -247,6 +250,16 @@ class Event
   void set_weight( weight t );
 
   /**
+   * Set drift_factor of the event (see DiffusionEvent).
+   */
+  virtual void set_drift_factor( weight t ){};
```

Comment:
Should we use `drift_weight` and `diffusion_weight` as names, maybe? 
@terhorstd Discuss in VC.